cli/src/main.rs

```diff
@@ -9,20 +10,25 @@ fn main() {
         .version(crate_version!())
         .author(crate_authors!())
         .setting(AppSettings::ArgRequiredElseHelp)
-        .subcommand(login_sub_command());
+        .subcommand(login_sub_command())
+        .subcommand(package_sub_command());
 
     let matches = app.get_matches();
     match matches.subcommand_name() {
         Some("login") => println!("loginnnn",), // login
+        Some("package") => package::create_package("./".to_string(), "/tmp/package.tar.gz".to_string()), // login
```

Comment:
was used just to run this, eventually the publish command will use create_package

Reply:
Just some terminology nit picking, is a package the tar? or is that the artifact? how these concept map to each other in your opinion?

Reply:
changed to artifact